material_theme_builder/lib/adaptive.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 The Flutter team. All rights reserved.
```

Comment:
This file is not used, do we still intend to have it here?

Reply:
Deleted.